### ISSUE TITLE
Welcome App shows Camunda in "Documentation"

### ISSUE THREAD

**Issue by kthoms** [operaton/operaton#363]
**Describe the bug**
The screenshot for "Documentation" is outdated. 

**To Reproduce**
Build/download a distro, and start it.

**Expected behavior**
An updated screenshot is shown.

**Scree
![Screenshot 340](https://github.com/user-attachments/assets/374438d8-1016-421a-bddb-6e58bbe6b9c1)
nshots**

**Comment by javahippie** [comment]
I think @lambdaschmied2 left that in when changing the others because we do not yet have a documentation page?

**Comment by lambdaschmied2** [comment]
Correct. Nothing to screenshot, but I'm having an eye on @a-m-zill and how he's coming along with the new docs site.
